services/blockchain-indexer/shared/dataService/transactions.js

```diff
@@ -95,7 +95,7 @@ const postTransactions = async params => {
 		const response = await dataService.postTransactions(params);
 		return {
 			message: 'Transaction payload was successfully passed to the network node',
-			transactionId: response.transactionId,
+			transactionID: response.transactionId,
```

Comment:
Please confirm if it is `response.transactionID` instead of `response.transactionId`.
```suggestion
			transactionID: response.transactionID,
```

Reply:
Yes it is `transactionId` in response from `lisk-sdk`